diff --git a/CMakeLists.txt b/CMakeLists.txt
index e6d8b1a..60b1813 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -125,8 +125,7 @@ endif()
 
 # Configure, gzip and install man pages
 configure_file(${EXTRA_DIR}/man/clight.1 clight.1 @ONLY)
-execute_process(COMMAND gzip clight.1 WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/clight.1.gz DESTINATION ${CMAKE_INSTALL_FULL_DATADIR}/man/man1)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/clight.1 DESTINATION ${CMAKE_INSTALL_FULL_DATADIR}/man/man1)
 
 #
 # Packaging support
